src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+import s from './MyPosts.module.css';
+import Post from './Post/Post';
+
+const MyPosts = () => {
+    return (
+        <div className={s.posts}>
+            <Post message='Hi, how are you' />
```

Comment:
Consider storing messages in separate const strings.

Reply:
I will do this in future versions of the app.